test/02LessonTwoTest.js

```diff
@@ -7,13 +7,23 @@ function whichSpecies(character) {
     // should return "cat" when character is 'garfield'
     // should return "fish" when character is 'nemo'
     // should return false if character is anything else
-
-}
+    if (character === 'scooby') {
+      return 'dog';
+    } else if (character === 'garfield') {
+      return 'cat';
+    } else if (character === 'nemo') {
+      return 'fish';
+    } else {
+      return "false";
+    }
 
```

Comment:
You're missing a closing curly brace here. As these stand, the tests crash because of this missing character.

```javascript
function whichSpecies(character) {
    // should return "dog" when character is 'scooby'
    // should return "cat" when character is 'garfield'
    // should return "fish" when character is 'nemo'
    // should return false if character is anything else
    if (character === 'scooby') {
      return 'dog';
    } else if (character === 'garfield') {
      return 'cat';
    } else if (character === 'nemo') {
      return 'fish';
    } else {
      return "false";
    }
}
```